dna/clutter/clutter.js

```diff
@@ -115,6 +115,25 @@ function newHandle(handle){
     return addHandle(handle);
 }
 
+// returns a map of user keys to handles
```

Comment:
This function neatly solves a rather ugly thing I was (and you were!) having to do to get all the handles for each post author that we had a hash for

However, what if the list of people who joined got really big? This response would bloat pretty big yes?

Reply:
Yes.It would, and in that kind of space this isn't the best solution.

Reply:
It could just be modified at some point to use pagination of some kind... which the main "meow" list is going to have to do something like as well

Reply:
Yes indeed, exactly.